.eslintrc.js

```diff
@@ -107,6 +107,36 @@ module.exports = {
         property,
         message: `Use ${alternative} instead.`,
       })),
+      ...[
+        'readdirSync',
+        'readFileSync',
+        'statSync',
+        'lstatSync',
+        'existsSync',
+        'pathExistsSync',
+        'realpathSync',
+        'mkdirSync',
+        'mkdirpSync',
+        'mkdirsSync',
+        'writeFileSync',
+        'writeJsonSync',
+        'outputFileSync',
+        'outputJsonSync',
+        'moveSync',
+        'copySync',
+        'copyFileSync',
+        'ensureFileSync',
+        'ensureDirSync',
+        'ensureLinkSync',
+        'ensureSymlinkSync',
+        'unlinkSync',
+        'removeSync',
+        'emptyDirSync',
+      ].map((property) => ({
+        object: 'fs',
+        property,
+        message: 'Do not use sync fs methods.',
+      })),
```

Comment:
wonder if this could be simplified with this rule?

https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-sync.md

Reply:
That rule only forbids `fs` methods, but we have `fs-extra` methods as well here.